README.md

```diff
@@ -1102,6 +1103,33 @@ frame.at_css("//a[text() = 'Log in']") # => Node
 #### select
 
 
+## Tracing
+
+You can use `tracing.start` and `tracing.stop` to create a trace file which can be opened in Chrome DevTools or [timeline viewer](https://chromedevtools.github.io/timeline-viewer/).
+
+```ruby
+browser.tracing.start(path: "trace.json")
+browser.go_to("https://www.google.com");
+browser.tracing.stop
+```
+
+#### tracing.start(trace_params:, \*\*options) : `Hash` | `Hash`
+
+* trace_params `Hash` 
+  * Specify [Tracing.start PARAMETERS](https://chromedevtools.github.io/devtools-protocol/tot/Tracing/#method-start).
+
+* options `Hash`
+  * `:path` (String) - `String` to save a Trace data output on the disk, not specified by default.
+  * `:encoding` (Symbol) - `:base64` | `:binary` setting only for memory Trace data output, `:binary` by default.
+  * `:screenshots` (Boolean) - When true - Captures screenshots in the trace, `false` by default.
+
+Only one trace can be active at a time per browser.
```

Comment:
So it's only one trace per browser but we delegate trace method to a page?

Reply:
I think the timeout error is raising because `Concurrent::Promises.resolvable_future` creates an object with the same object_id, that already has a subscription on the `Tracing.tracingComplete` event from the previous test.